code/modules/projectiles/projectile/bullets.dm

```diff
@@ -53,6 +53,18 @@
 /obj/item/projectile/bullet/heavybullet
 	damage = 35
 
+/obj/item/projectile/bullet/sabot
+	damage = 150
+	var/breakthings = FALSE

```

Comment:
whoops. I removed breakthings = TRUE. I should readd that or remove this. One of the two.